kt/godot-library/src/main/kotlin/godot/core/bridge/Callable.kt

```diff
@@ -4,7 +4,7 @@ package godot.core
 import godot.Object
 import godot.core.callable.KtCallable
 
-interface Callable {
+interface Callable: CoreType {
```

Comment:
So I assume now Callables cannot be lateinit ?

Reply:
no callables are not allowed to be lateinit if registered